index.bs

```diff
@@ -536,6 +536,25 @@ i.e. consistency within CSS.
 
 There is also a separate section on <a href="#naming-consistency">naming consistency</a>.
 
+<h3 id="specs-include-guidance">Follow guidance from feature specifications</h3>
+
+When using a feature,
+follow guidance in the specification for that feature.
+
+Specifications for features
+that will be used by other specifications
+should include guidance for using those features.
+This ensures that the feature is used correctly
+and consistently
+throughout the platform.
+
+An incomplete list of such guidance follows:
+
+* [[encoding#specification-hooks|Encoding]]
+* [[fetch#fetch-elsewhere|Fetch]]
+* [[hr-time#sec-tools|Time]]
+* [[url#url-apis-elsewhere|URLs]]
+
```

Comment:
```suggestion
*  [[#using-http]]

```

Reply:
I've added this, but I wonder if, in the interest of conciseness, we should put
> [Building Protocols with HTTP](https://httpwg.org/specs/rfc9205.html) and [other HTTP RFCs](https://datatracker.ietf.org/doc/search?name=HTTP&sort=&rfcs=on&by=group&group=httpbis)

here and remove the #using-http section.